lib/providers/providers.dart

```diff
@@ -20,7 +20,7 @@ final providers = [
     update: (_, auth, http) => http
       ..init(
         accountName: auth.accountName,
-        enableMockResponse: false,
+        enableMockResponse: true,
```

Comment:
That's for development purpose, but noticed that for automated .apk release should be commited with false flag in repository